src/utils/stripUnwantedTags.js

```diff
@@ -0,0 +1,20 @@
+// utils/srtipUnwantedTags
+
+import { parseHTML } from 'linkedom'
+
+/**
+ * @param html {string}
+ * @param exclusions {string[]}
+ * @returns {string}
+ */
+export default (html, exclusions = []) => {
+  if (typeof exclusions !== 'undefined' && !exclusions.length) return html
+
+  const { document } = parseHTML(html)
```

Comment:
Use the `new DOMParser().parseFromString(html, 'text/html')`. `DOMParser` will be the native class when running in the browser.
https://github.com/ndaidong/article-parser/blob/main/src/browser/linkedom.js
The polyfills of linkedom only export the `DOMParser` now